src/runtime/virtcontainers/hypervisor.go

```diff
@@ -399,6 +399,10 @@ type HypervisorConfig struct {
 	// The PCIe Root Port device is used to hot-plug the PCIe device
 	PCIeRootPort uint32
 
+	// PCIeLazyAttachDelay is a sandbox annotation used to indicate if devices whether to hot-plug
```

Comment:
If this configuration supports using annotation, it's better update https://github.com/kata-containers/kata-containers/blob/2.0-dev/docs/how-to/how-to-set-sandbox-config-kata.md at the same time.

Reply:
OK.

Reply:
Added annotation "io.katacontainers.config.hypervisor.pcie_lazy_attach_delay" to then document.